mxcube3/routes/Queue.py

```diff
@@ -383,6 +384,12 @@ def add_centring(id):
     return resp
 
 
+@mxcube.route("/mxcube/api/v0.1/results/<int:number>")
+def serve_char_result(number):
```

Comment:
as I said, how to present the result is left out of this pr, even if those lines were there. In any case I removed those